design/sd-local.md

```diff
@@ -141,6 +141,7 @@ $ sdlocal build [job-name] [options]
 - `--src-url [repository url]` Set repository URL which is to build when user use the remote repository without local files.
 - `--disable-image-pull` Disable `sd-local` from always pulling build image.
 - `--sudo` Use `sudo` command to execute docker runtime.
+- `--local` Use `.sdlocal/config` in current directory as a config file of sd-local.
```

Comment:
I think it's better to unify with the description in `config` subcommand.

Reply:
The description of `build` subcommand has 3 different `config` for `meta` , `env` and `sd-local`.
Therefore, I worry that unifying makes users confused.

Do you have anything good ideas?